lib/gax.js

```diff
@@ -90,6 +76,59 @@ function CallSettings(settings) {
 exports.CallSettings = CallSettings;
 
 /**
+ * Encapsulates the overridable settings for a particular API call.
+ *
+ * ``CallOptions`` is an optional arg for all GAX API calls.  It is used to
```

Comment:
Can we provide an example of how typical use of a GAX API call combines optional API call parameters with `CallOptions` parameters?